Dockerfile_mbtci_java8

```diff
@@ -0,0 +1,118 @@
+# On `22.04 LTS (Jammy Jellyfish)` the `apt-get` command fails
+# https://ubuntu.com/about/release-cycle
+FROM ubuntu:focal
+
+# Build time arguments
+ARG DEBIAN_FRONTEND=noninteractive
+ARG MTA_USER_HOME=/home/mta
+ARG MBT_VERSION=1.2.18
+ARG GO_VERSION=1.15.6
+# node version as found in https://nodejs.org/dist/ e.g: "v16.15.0"
+ARG NODE_VERSION=NODE_VERSION_TEMPLATE
+ARG MAVEN_VERSION=3.6.3
+# This is used to build the download URL
+# e.g: https://tools.hana.ondemand.com/additional/sapjvm-8.1.086-linux-x64.zip
+# note that sometimes old versions are removed and new patch versions are added
+# so this argument's value may need to be updated in the future
+# Also see JVM section here:
+# - https://tools.hana.ondemand.com/#cloud
+ARG JAVA_VERSION=8.1.086
+
+# Environment variables
+ENV JAVA_HOME /opt/jdk
+ENV MAVEN_HOME /usr/apache-maven-$MAVEN_VERSION
+ENV PATH $PATH:$MAVEN_HOME/bin:${JAVA_HOME}/bin
+ENV PYTHON /usr/bin/python3
+ENV M2_HOME=/opt/maven/apache-maven-${MAVEN_VERSION}
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
+ENV CGO_ENABLED=0
+ENV GOOS=linux
+ENV DEBIAN_FRONTEND=noninteractive
+
+ADD http://aia.pki.co.sap.com/aia/SAP%20Global%20Root%20CA.crt \
+    /etc/ssl/certs/SAP_Global_Root_CA.crt
+
+# Download required env tools
+RUN apt-get update && \
+    apt-get install --yes --no-install-recommends \
+		ca-certificates \
+		git \
+		make \
+		curl \
+		wget \
+		unzip && \
+
+    # Change security level as the SAP npm repo doesnt support buster new security upgrade
+    # the default configuration for OpenSSL in Buster explicitly requires using more secure ciphers and protocols,
+    # and the server running at http://npm.sap.com/ is running software configured to only provide insecure, older ciphers.
+    # This causes SSL connections using OpenSSL from a Buster based installation to fail
+    # Should be remove once SAP npm repo will patch the security level
+    # see - https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=912759
+    sed -i -E 's/(CipherString\s*=\s*DEFAULT@SECLEVEL=)2/\11/' /etc/ssl/openssl.cnf && \
+
+# install java
+	mkdir -p /opt && \
+    wget --no-check-certificate --no-cookies --header "Cookie: eula_3_1_agreed=tools.hana.ondemand.com/developer-license-3_1.txt; path=/;" -S https://tools.hana.ondemand.com/additional/sapjvm-${JAVA_VERSION}-linux-x64.zip && \
```

Comment:
Security issue: it's not acceptable to not check the certificate chain of a binary distribution endpoint.